tests/e2e.sh

```diff
@@ -24,7 +24,7 @@ case $(uname -m) in
 esac
 
 NODE_IMAGE_NAME="docker.io/kindest/node"
-KUBERNETES_VERSION=v1.20.0
+KUBERNETES_VERSION=${KUBERNETES_VERSION:-v1.20.0}
```

Comment:
We might want to change the default to `v1.21.0` now that it has been released.

Reply:
Done thanks!